index.html

```diff
@@ -3870,12 +3871,60 @@
   const index = keyboardFocus.caretMesh.getValue();
   keyboardFocus.textMesh.setText(s.slice(0, index) + c + s.slice(index));
   keyboardFocus.caretMesh.setValue(index + 1);
+
+  _autoCompleteSearch(keyboardFocus.textMesh.getText());
```

Comment:
Do we want to always autocomplete on every single mode of the label mesh? In particular I think it might not make sense for the 2D browser mesh case.